IFC4x3/Properties/t/Thickness_2wgAijH9HBAhOdWTCDnu2Y/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Thickness" UniqueId="baa8ab2d-4494-4b2a-b627-81d30dc780a2" PrimaryDataType="IfcPositiveLengthMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Thickness_2wgAijH9HBAhOdWTCDnu2Y" Name="Thickness" UniqueId="baa8ab2d-4494-4b2a-b627-81d30dc780a2" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Is Thickness an actual measure in addition to the NominalThickness? Isn't it then a quantity?